### ISSUE TITLE
Each word in hashtags should be capitalized

### ISSUE THREAD

**Issue by numeratorjik** [simonfrey/mastodon_wordpress_autopost#63]
The plugin only capitalizes the first word when converting tags and categories into hashtags. For accessibility reasons, each word in the hashtag should be capitalized. For example, if there's a tag or category called "information security", then the hashtag should be "#InformationSecurity", not "#Informationsecurity".
